package.json

```diff
@@ -15,9 +24,15 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.5.0",
-    "eslint-plugin-react": "^7.5.1"
+    "eslint-plugin-react": "^7.5.1",
+    "prettier": "^1.10.2"
   },
   "peerDependencies": {
     "eslint": "^4.16.0"
+  },
+  "devDependencies": {
```

Comment:
does eslint need to be a peerDependency and a devDependency?

Reply:
Yes it's two different use cases - the `peerDependencies` version is for the consuming app, and `devDependencies` is purely for linting and fixing this project when you work on it.